src/accordion.tsx

```diff
@@ -39,9 +42,11 @@ interface AccordionItemContentProps {
     subtitle?: string;
     asset?: React.ReactNode;
     content: React.ReactNode;
-    onToogle?: (value: boolean) => void;
     dataAttributes?: DataAttributes;
     trackingEvent?: TrackingEvent | ReadonlyArray<TrackingEvent>;
+    role?: string;
+    /** @deprecated Use onChange Accordion's onChange callback instead */
+    onToogle?: (value: boolean) => void;
```

Comment:
This callback had no effect and it's not used anywhere. Given that there are no usages of accordion in webapp, I propose deleting it to avoid confusions. For now, I deprecated it. WDYT?

Reply:
please, don't remove it, just deprecate it. It would be a breaking change. Even if it isn't used in Novum webapp it could be used by other clients